### ISSUE TITLE
Villfisk

### ISSUE THREAD

**Issue by Hypsibius** [realfagstermer/realfagstermer#513]
Foreslår ny term Villfisk 

https://bibsys-almaprimo.hosted.exlibrisgroup.com:443/UBB:default_scope:BIBSYS_ILS71479939880002201

Vi treng ikkje ein definisjon for den, gjer vi? Både SNL og Wikipedia brukar termen, men definerer ikkje.

**Comment by BioHeidi** [comment]
Feil url?

**Comment by danmichaelo** [comment]
> Feil url?

Funket for meg :S

**Comment by BioHeidi** [comment]
Det var klønete formulert. Jeg lurte bare på om det var feil bok det var lenket til, for jeg skjønte ikke imuddelbart hvorfor den skulle ha emneordet villfisk. Men sånn bortsett fra det er kanskje villfisk en ok term, bare den ikke blir brukt på alt mulig (alt som ikke er oppdrettsfisk er villfisk, så hvis det står oppdrettsfisk på det som ikke er villfisk så behøver vi strengt tatt ikke villfisk-term?)

**Comment by Hypsibius** [comment]
Indeed! Godt sett, Heidi! Antar den eigentlege lenka var avhandlinga eg behandla før eller etter :-/ Men kva for ei var DET da!

Og enig, vil(l) ikkje brukje omgrepet om alt som ikkje er oppdrettsfisk, default er at ein fisk er vill. Behov for termen når ein spesifiserer eksempelvis villaks/-ørret mot rømt/utsett, eller smitte frå mærd til villfisk etc.

**Comment by bibliomari** [comment]
Betyr det at dere vill ha Villfisk? Med definisjon/note formulert i retning av det @Hypsibius sier? @BioHeidi

**Comment by Hypsibius** [comment]
Tjah, vil ha, men klarer ikkje å finne belegget. Vil dokke ventet til eg finn eit passande?

**Comment by danmichaelo** [comment]
Det hadde ikke vært dumt, @Hypsibius  

🐟🐟🐟
